pytorch_lightning/trainer/training_loop.py

```diff
@@ -16,6 +16,7 @@
 from copy import copy, deepcopy
 
 import numpy as np
+from numpy.lib.arraysetops import isin
```

Comment:
where's this used?

Reply:
it's not. was added by my IDE as auto import during typing `isinstance` :D
removed it